Source/Scene/Scene.js

```diff
@@ -2696,6 +2706,68 @@ define([
         return object;
     };
 
+    var scratchPickDepthPassState;
+
+    function renderForPickDepth(scene, drawingBufferPosition) {
+        var context = scene._context;
+        var us = context.uniformState;
+        var frameState = scene._frameState;
+
+        // Update with previous frame's number and time, assuming that render is called before picking.
+        updateFrameState(scene, frameState.frameNumber, frameState.time);
+        frameState.cullingVolume = getPickCullingVolume(scene, drawingBufferPosition, 1, 1);
+        frameState.passes.render = true;
```

Comment:
The shadow map pass happens during the render pass. It uses derived commands that disables color writes and enables depth writes. Also, the framebuffer only has a depth attachment.

I could change it to pick, but the depth copies for each frustum only happen during the render pass. Another option is to add a new depth pass.